test/gie/more_builtins.gie

```diff
@@ -243,6 +243,23 @@ expect    failure errno no_args
 
 
 
+-------------------------------------------------------------------------------
+Tests for PJ_lccm.c
+-------------------------------------------------------------------------------
+This test is taken from EPSG guidance note 7-2 (version 54, August 2018,
```

Comment:
Is this the only test coordinate in guidance note 7-2? If so, can you dig out some coordinates from another source?

Reply:
Yes, it's the only one test in that document. I'll try to google that projection and see if I can find any other test coordinates. But I think just one serious test for the common case is enough (other "normal" coordinates probably won't add much in terms of testing if they are not corner cases)

Reply:
I've tried to find more "official" test coordinates, but haven't found any. While doing my research I've found this very relevant and interesting message:

http://lists.maptools.org/pipermail/proj/2015-April/007108.html

It provides three ways of emulating 2SP Michigan in existing proj.4. Author of the message provides several coordinates which he used for his testing, but he warns that they are not official and he may have made a mistake somewhere. 

However, what is more interesting - is that one can just use normal `k_0` option instead of my new `scale` and it magically works! I've just tried it in my tests and indeed, if we just replace `+scale=1.0000382` with `+k_0=1.0000382` it still gives the same results for the official test coordinates! Even though it obviously hits the different code path.

Reply:
Actually it is not magic at all. When looking closely at the formulas, I can convince myself that k_0 incorporates the effect of the new scale parameter. In theory we could merge both together, as LCC_2SP Michigan doesn't use the k_0 = scale factor at natural origin which is used only by LCC_1SP. In practice, it might be good to have the different parameters to distinguish the different semantics (and k_0 is typically < 1, whereas scale is typically > 1). I don't have a strong opinion on this however

Reply:
Are there scenarios where you would use both the `+k_0` and `+scale` factors?

If not I prefer to reuse `+k_0`. It's the canonical way to define a scale factor in PROJ and has been since forever. Care should be taken to explain in the examples how it can be used in different circumstances.

Reply:
> I am not sure about scenarios of simultaneous use of both `+k_0` and `+scale` factors

Neither LCC_1SP, LCC_2SP or LCC_2SP_Michigan require both, so we can probably forget about tose

> Frankly, I feel somewhat disheartened by this issue

Sorry for that. We don't really know about the real difficulty (or triviality!) unless really digging into things. That wasn't mean to trap you. Your work on this has been very helpful, at least to demistify LCC_2SP Michigan... If you could just remove the scale parameter in the PJ_lcc.c file and replace scale by k_0 in the documentation and tests, I think your PR should be good to merge !

Reply:
> Frankly, I feel somewhat disheartened by this issue.

I am sorry to hear that. To be honest, I assumed that this would need to be a new projection in it's own right when I wrote that issue. You have helped clear that up, so your efforts haven't been in vain. Thank you for that.

I still feel like this is a "good first issue". Turns out that it is rather simple to fix :-) But in all seriousness, I somewhat expect a bit of dialogue before new contributors take on issues. Not that you aren't allowed to just crack on but sometimes we can help the process along from the start so a bunch of unnecessary work is avoided. In this case we would probably have realised that there's wasn't much else to this issue than documentation.

I hope you aren't completely discouraged and will consider contributing on other issues. There are plenty of them, even for programmers that don't know much about geography and cartography :-)

Reply:
@rouault @kbevers Thank you for your support and prompt responses, much appreciated.

> If you could just remove the scale parameter in the PJ_lcc.c file and replace scale by k_0 in the documentation and tests, I think your PR should be good to merge !

Ok, I've done that and updated everything related to `+scale`. I have also explicitly included `+k_0` in the LCC documentation providing slightly different, more specific description for this projection, mentioning SP2 Michigan. 

> I somewhat expect a bit of dialogue before new contributors take on issues. 

Makes sense. Before I started I hadn't expected that the maintainers of the package would respond that quickly, so I thought it's better to do something first and then start discussing to make the discussion more concrete :)

> I hope you aren't completely discouraged and will consider contributing on other issues. 

Haha, yes, I am tempted by the famous Dymaxion, but it looks somewhat scary :)